src/main/cms/cms_menu_gps_rescue.c

```diff
@@ -216,8 +216,8 @@ const OSD_Entry cmsx_menuGpsRescueEntries[] =
     { "DESCENT RATE CM/S", OME_UINT16 | REBOOT_REQUIRED, NULL, &(OSD_UINT16_t){ &gpsRescueConfig_descendRate, 25, 500, 1 } },
     { "LANDING ALT     M", OME_UINT8 | REBOOT_REQUIRED, NULL, &(OSD_UINT8_t){ &positionControlConfig_landingAltitudeM, 1, 15, 1 } },
 
-    { "THROTTLE MIN",      OME_UINT16 | REBOOT_REQUIRED, NULL, &(OSD_UINT16_t){ &gpsRescueConfig_throttleMin, 1000, 2000, 1 } },
-    { "THROTTLE MAX",      OME_UINT16 | REBOOT_REQUIRED, NULL, &(OSD_UINT16_t){ &gpsRescueConfig_throttleMax, 1000, 2000, 1 } },
+    { "THROTTLE MIN",      OME_UINT16 | REBOOT_REQUIRED, NULL, &(OSD_UINT16_t){ &positionControlConfig_throttleMin, 1000, 2000, 1 } },
+    { "THROTTLE MAX",      OME_UINT16 | REBOOT_REQUIRED, NULL, &(OSD_UINT16_t){ &positionControlConfig_throttleMax, 1000, 2000, 1 } },
```

Comment:
```suggestion
    { "THROTTLE MIN",      OME_UINT16 | REBOOT_REQUIRED, NULL, &(OSD_UINT16_t){ &positionControlConfig_throttleMin, 1050, 1400, 10 } },
    { "THROTTLE MAX",      OME_UINT16 | REBOOT_REQUIRED, NULL, &(OSD_UINT16_t){ &positionControlConfig_throttleMax, 1400, 2000, 10 } },
```

Reply:
Yes, probably, and remove them from the user configuration, and I mentioned above.

That can be done in a separate PR since it would require a separate discussion.

This is a refactoring PR.

Reply:
I don't know if we are talking cross-purposes. With the suggestion I want to point out a small bug: In the CMS menu you can enter invalid values for throttleMin and throttleMax.
The comment below about macros is the solution to prevent this from happening again (code refactoring)

Reply:
My apologies, indeed I misunderstood :-)
I understand now.  
I'll make the CMS range the same as the CLI.
Later on I will address if we really need user limits at all.
Thanks.